libs/perun/components/src/lib/perun-footer/perun-footer.component.ts

```diff
@@ -46,23 +47,37 @@ export class PerunFooterComponent implements OnInit {
   headersTextColor: string = this.storeService.get('theme', 'footer_headers_text_color') as string;
   linksTextColor: string = this.storeService.get('theme', 'footer_links_text_color') as string;
   githubRepository: string = this.storeService.get('footer', 'github_releases') as string;
+  githubBackendRepository: string = this.storeService.get(
+    'footer',
+    'github_backend_releases'
+  ) as string;
   iconColor: string = this.storeService.get('theme', 'footer_icon_color') as string;
   bgColor: string = this.storeService.get('theme', 'footer_bg_color') as string;
-  version = '';
+  backendVersion = '';
+  guiVersion = '';
   language = 'en';
   columnContentHeight = 0;
 
   constructor(
     private storeService: StoreService,
     private translateService: TranslateService,
+    private utilsService: UtilsService,
     private dialog: MatDialog
   ) {}
 
   ngOnInit(): void {
     this.translateService.onLangChange.subscribe((lang) => {
       this.language = lang.lang;
     });
-    this.version = require('../../../../../../package.json').version as string;
+
+    this.guiVersion = require('../../../../../../package.json').version as string;
+    this.utilsService.getPerunStatus().subscribe((val) => {
+      const versionString = val[0];
+      this.backendVersion = versionString.substring(versionString.indexOf(':') + 2);
+      this.backendVersion = '17.2.0';
+      this.guiVersion = '17.2.0';
```

Comment:
You probably forgot to delete these two lines.
```suggestion
```

Reply:
Yes, thank you for pointing it out.